editor/editor.css

```diff
@@ -131,7 +131,15 @@
     height: 100%;
 }
 
-.side-panel .side-panel-button {
+.side-subpanel {
+    padding-top: 5px;
+    background-color: #fff;
```

Comment:
Maybe, use a darker color instead? I feel using a darker color instead of white is more natural for deeper hidden menus. But, this might be just a personal preference.

```suggestion
    background-color: #F1F1F1;
```

**Screenshot:**
![Screen Shot 2022-06-07 at 17 49 38](https://user-images.githubusercontent.com/9922882/172489672-e03b9d38-dda8-4c54-bc87-09b96fb7aa0a.png)